compression/tests/CompressionDictionary_unittest.cpp

```diff
@@ -2274,10 +2274,10 @@ TEST_P(CompressionDictionaryTest, VarCharKeyLimitCodesForComparisonDifferentType
 
 INSTANTIATE_TEST_CASE_P(WithAndWithoutNullValue,
                         CompressionDictionaryTest,
-                        ::testing::Bool());
+                        ::testing::Bool(),);  // NOLINT(whitespace/comma)
```

Comment:
Can you add a comment here about why the comma is needed (Clang compilation fix), so that in the future people can benefit when they write similar code.